environments/liveness/liveness-environment/amplify/backend/function/livenessenvironment09873267/src/index.js

```diff
@@ -0,0 +1,40 @@
+import { RekognitionLiveness } from '@amzn/aws-rekognition-liveness-js-v2-client';
+
+/**
+ * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
+ */
+
+export const handler = async (event, req) => {
+  console.log({ req });
+  console.log({ event });
+
+  const client = new RekognitionLiveness({
+    region: 'us-east-1',
+    endpoint: `https://us-east-1.gamma.frontend.reventlov.rekognition.aws.dev`,
```

Comment:
Sorry if this is late notice but I'm realizing we could probably parameterize the `region` and `endpoint` values to be passed in so that we don't need to create a whole new lambda function?

Reply:
Discussed offline, probably not feasible rn because the lambdas also consume different SDKs